### PR TITLE
Add Tests and fix calculations

### PR DESCRIPTION
**Pull request by adamskrodzki**

### CODE REVIEW

File: packages/multiply/src/calculateMultParams.ts [hunk 1]
```diff
@@ -185,22 +187,14 @@ function getCloseToDaiParams(
   skipFL: boolean;
 } {
   const _skipFL = false;
-  const maxCollNeeded = vaultInfo.currentDebt
-    .times(1.00001 /* to account for not up to date value here */)
```
Comment (ferni) [inline packages/multiply/src/calculateMultParams.ts:189]: Could you expand on this?

Comment (adamskrodzki) [reply packages/multiply/src/calculateMultParams.ts:189]: this is due to the fact that moment when transaction is mined pight be several blocks later than when transaction parameters where calculated and Maker Protocol accumulated new fees. 

same debtOffset can be found in current oasis-borrow and this is replacement for it

Comment (ferni) [reply packages/multiply/src/calculateMultParams.ts:189]: Ok cool, looks good.

File: packages/multiply/test/close-to-collateral.test.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    expect(result.toTokenAmount.toNumber()).to.be.equal(103217.55484563917);
+  });
+});
+describe('getCloseToCollateralParams skipFF = false', async () => {
+  beforeEach(async () => {
+    marketParams = new MarketParams({
+      marketPrice: 3000,
+      oraclePrice: 3000,
+      FF: 0.0009,
+      OF: 0.0003,
```
Comment (sennett) [inline packages/multiply/test/close-to-collateral.test.ts:39]: What are these?

File: packages/multiply/test/close-to-collateral.test.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      slippage: 0.03,
+    });
+    vaultInfo = new VaultInfo(100000, 83, 1.5);
+  });
+
+  it('should return correct toTokenAmount and requiredDebt', async () => {
+    const result = getCloseToCollateralParams(marketParams, vaultInfo);
+    expect(result.withdrawCollateral.toNumber()).to.be.equal(0);
+    expect(result.skipFL).to.be.equal(false);
```
Comment (sennett) [inline packages/multiply/test/close-to-collateral.test.ts:48]: What is FL?

File: packages/multiply/src/calculateMultParams.ts [hunk 4]
```diff
@@ -6,6 +6,8 @@ import {
   calculateParamsDecreaseMP,
 } from './internal/increaseDecreaseMP';
 
+export const OFFSET_MULTIPLAYER = new BigNumber(1.00001);
```
Comment (sennett) [inline packages/multiply/src/calculateMultParams.ts:9]: Could this be a env var?

**Review by sennett — COMMENTED** [review]
I can't comment on the maths, but the rest of it looks good.

**Comment by sennett** [comment]
Lol it was already merged.

**Pull request merged**